src/theme/icons/plus.ts

```diff
@@ -0,0 +1,20 @@
+/*!
+ * Font Awesome Free 5.15.0 by @fontawesome - https://fontawesome.com
+ * License - https://fontawesome.com/license/free (Icons: CC BY 4.0, Fonts: SIL OFL 1.1, Code: MIT License)
+ */
+import {
+  IconDefinition,
+  IconName,
+  IconPrefix,
+} from '@fortawesome/fontawesome-svg-core';
+
+export const width = 448;
+export const height = 512;
+export const unicode = 'e010';
```

Comment:
:) it's hexadecimal, so next number after `e009` is not `e010` but `e00a`. I got also caught  :D

Reply:
oh man, good catch! I even didn't stop to think about it